blueprints/modular/intro.yaml

```diff
@@ -0,0 +1,36 @@
+title: Intro
+'@extends': default
+
+form:
+  fields:
+    tabs:
+      fields:
+        buttons:
+          type: tab
+          title: Intro
+          fields:
```

Comment:
Any reason why you didn't add `header.links`?

Reply:
Must have slipped my mind. I have already implemented it for myself, I think its not a part of this branch. Will add.